src/components/Comments/WriteComment.js

```diff
@@ -190,7 +190,7 @@ export const styles = StyleSheet.create({
     backgroundColor: COLORS.white,
     width: '100%',
     paddingRight: 20,
-    paddingLeft: inReplyCommentView ? 50 : 20,
+    paddingLeft: inReplyCommentView ? 43 : 20,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in paddingLeft from 50 to 43 when `inReplyCommentView` is true seems minor, but it could potentially affect the layout of your application. Ensure that this change doesn't cause any misalignment or overlap with other UI elements.
```diff
-    paddingLeft: inReplyCommentView ? 50 : 20,
+    paddingLeft: inReplyCommentView ? 43 : 20,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->